winpr/libwinpr/file/file.c

```diff
@@ -424,7 +424,7 @@ static HANDLE FileCreateFileA(LPCSTR lpFileName, DWORD dwDesiredAccess, DWORD dw
 	WINPR_FILE* pFile;
 	BOOL create;
 	const char* mode = FileGetMode(dwDesiredAccess, dwCreationDisposition, &create);
-	int lock;
```

Comment:
I think this lock is also always set when it is used (either if FILE_SHARE_READ or FILE_SHARE_WRITE is set in dwShareMode)